integration-tests/test-cases/scenario-tests/src/test/java/io/cellery/integration/scenario/tests/petstore/PetStoreTestCase.java

```diff
@@ -0,0 +1,230 @@
+     */
+    @Test
+    public void checkoutCartAlice() throws InterruptedException {
+        this.cart.checkout();
+        TimeUnit.SECONDS.sleep(10);
+        validateWebPage("0", this.cart.getNumberOfItems(), "Pet store cart content after checkout is " +
+                "not as expected");
+    }
+
+    /**
+     * This tests check order in for user Alice.
+     * @throws InterruptedException
+     */
+    @Test
+    public void checkOrdersAlice() throws InterruptedException {
+        TimeUnit.SECONDS.sleep(10);
+        validateWebPage("$ 675.00", this.order.getOrderValue(), "Pet store cart content after checkout is " +
+                "not as expected");
+    }
+
+    /**
+     * This tests sign out for user Alice.
+     */
+    @Test
+    public void signOutAlice() {
+        this.signOut(this.alice);
+    }
+
+    /**
+     * This tests sign in for user Bob.
+     * @throws InterruptedException
+     */
+    @Test
+    public void signInBob() throws InterruptedException {
+        this.signIn(this.bob);
+    }
+
+    /**
+     * This tests check order in for user Bob.
+     * @throws InterruptedException
+     */
+    @Test
+    public void checkOrdersBob() throws InterruptedException {
+        TimeUnit.SECONDS.sleep(10);
+        validateWebPage(Constants.PET_STORE_NO_ORDERS_PLACED, this.order.getOrderValue(),
+                "Pet store cart content after checkout is not as expected");
+    }
+
+    /**
+     * This tests sign out for user Bob.
+     */
+    @Test
+    public void signOutBob() {
+        this.signOut(this.bob);
+    }
+
+    @Test
+    public void terminate() throws Exception {
+        terminateCell(backEndInstanceName);
+        terminateCell(frontEndInstanceName);
+    }
+
+    @AfterClass
+    public void cleanup() {
+        webDriver.close();
+        try {
+            terminateCell(backEndInstanceName);
+            terminateCell(frontEndInstanceName);
+        } catch (Exception ignored) {
+        }
+    }
+
+    public void signIn(User user) throws InterruptedException {
+        String singInHeader = user.clickSignIn();
+        validateWebPage(singInHeader, Constants.PET_STORE_SIGN_IN_WEB_CONTENT, "Pet store sign in web page " +
+                "content is not as expected");
+
+        String personalInfoHeader = user.submitCredentials();
+        validateWebPage(personalInfoHeader, Constants.IDENTITY_SERVER_HEADER, "Identity server web page " +
+                "content is not as expected");
+
+        user.acceptPrivacyPolicy();
+
+        String petAccessoriesHeader = user.submitInformation();
+        validateWebPage(petAccessoriesHeader, Constants.PET_STORE_PERSONAL_INFORMATION_HEADER, "Pet store sign " +
+                "in web page content is not as expected");
+    }
+
+    public void signOut(User user) {
+        String idpLogoutHeader = user.signOut();
+        validateWebPage(idpLogoutHeader, Constants.PET_STORE_IDENTITY_SERVER_LOGOUT_HEADER, "IDP logout header " +
```

Comment:
Is this Constants.PET_STORE_IDENTITY_SERVER_LOGOUT_HEADER going to be used by multiple test classes? If not, you can specify as simply class level attribute.

Reply:
Updated as PetStoreTestCase class attributes.